### PR TITLE
Update homebrew installer and use ed25519 for github keys.

### PR DESCRIPTION
**Pull request by alexpertsev**
## Why?

Bootstrap script was failing to install homebrew as the ruby homebrew installer has been disabled.
Switched it to recommended command. 
Also had issues with the current rsa algorithm for github keys, github recommends ed25519 as default and rsa
for legacy systems.

## Merge Instructions

Please **DO** squash my commits when merging

### CODE REVIEW

**Comment by o-seismic** [comment]
Hi @alexpertsev, I'm new here. I ran bootstrap script of main branch. They're broken.
I'm not sure why this PR is not merged for now.
One more thing is that 'brew "postgresql"' would be failed. It needs a specific version.

Screenshot 2023-03-28 at 16 37 56

If possible, please merge your PR to main branch.

**Comment by alexpertsev** [comment]
Hey @o-seismic, you can switch to this PR branch and run the script from there. Could you run and confirm it's working for you? Regarding postgresql, you can install `postgresql@13` locally. This is mainly needed for drivers for your postgresql client (e.g. pgAdmin, postico etc). For the application, postgresql will be running inside docker container.

File: bin/bootstrap [hunk 1]
```diff
@@ -86,7 +87,8 @@ if [[ $? != 0 ]]; then
   echo "Github SSH key not found. Generating..."
   echo -en "\n\033[1mEnter your the email address you'll use for Github:\033[0m "
   read email
-  ssh-keygen -f ~/.ssh/github -t rsa -b 4096 -C $email
+  # use recommended ed25519 over rsa
+  ssh-keygen -f ~/.ssh/github -t ed25519 -C $email
```
Comment (emmoistner) [inline bin/bootstrap:91]: This only runs for folks who have not setup github ssh yet, should we also add a path for folks to upgrade?

Comment (alexpertsev) [reply bin/bootstrap]: @emmoistner, updated file name to force ed25519 key creation.

File: bin/bootstrap [hunk 2]
```diff
@@ -81,26 +82,27 @@ fi
 #
 # Github ssh key setup
 #
-stat ~/.ssh/github
+stat ~/.ssh/id_ed25519
```
Comment (emmoistner) [inline bin/bootstrap:85]: Would it better to still label this for use with github? 

```suggestion
stat ~/.ssh/github_ed25519
```

Comment (alexpertsev) [reply bin/bootstrap]: Sure, renamed it

**Review by emmoistner — APPROVED** [review]
Looking good! I was able to even update my ssh key as well. 🙇

**Pull request merged**